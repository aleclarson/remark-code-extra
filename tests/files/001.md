```
simple code block
```

```ts
language code block
```