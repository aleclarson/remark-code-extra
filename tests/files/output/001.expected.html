<pre><code class="language-not-skipped">simple code block
</code></pre>
<pre><code class="language-skipped">language code block
</code></pre>
<pre><code>non-language block
</code></pre>
